crates/neon-runtime/src/napi/array.rs

```diff
@@ -1,13 +1,13 @@
 //! Facilities for working with Array `napi_value`s.
 
-use raw::{Env, Local};
+use crate::raw::{Env, Local};
 
-use nodejs_sys as napi;
+use crate::napi::bindings as napi;
 
 pub unsafe extern "C" fn new(out: &mut Local, env: Env, length: u32) {
     assert_eq!(
-        napi::napi_create_array_with_length(env, length as usize, out as *mut _),
-        napi::napi_status::napi_ok,
+        napi::create_array_with_length(env, length as usize, out as *mut _),
```

Comment:
These naming conventions aren't, like, gonna change the world, but they're so nice 😎

Reply:
What's not to love about repeating the word `napi` 3x in a single identifier. 😆  `s/napi::napi_status::napi_ok/napi::Status::Ok/`

napi, napi.... napi.

...napi